osu.Game/Screens/Edit/Compose/Components/EditorBlueprintContainer.cs

```diff
@@ -34,13 +34,6 @@ protected EditorBlueprintContainer(HitObjectComposer composer)
         [BackgroundDependencyLoader]
         private void load()
         {
-            // For non-pooled rulesets, hitobjects are already present in the playfield which allows the blueprints to be loaded in the async context.
```

Comment:
Nit: there is also a symmetrical comment in the other path a bit lower down.

https://github.com/ppy/osu/blob/d8d2c731ee42db1ecc7e1a6c8cb3d5f1dad8b943/osu.Game/Screens/Edit/Compose/Components/EditorBlueprintContainer.cs#L72

Should probably trim both to avoid confusion.